app/extensions/gitlab.py

```diff
@@ -22,6 +22,10 @@
 log = logging.getLogger(__name__)
 
 
+class GitlabInitializationError(RuntimeError):
```

Comment:
What is the benefit of an empty derived class?

Reply:
It is to give a specific error for us to catch.  `RuntimeError` can happen when it's totally unrelated to gitlab and I don't want to catch that.